test/test/mocks/widgets.js

```diff
@@ -1,3 +1,36 @@
+export const BODY_OK =
+{
+  payload: {
+    code: 'test_widget',
+    name: 'Test Widget',
+    used: 0,
+    titles: {
+      it: 'Widget di Test',
+      en: 'Test Widget',
+    },
+    group: 'administrator',
+    customUi: '<p>Custom UI</p>',
+    defaultUi: '<p>Default UI</p>',
+    createdAt: '2018/02/22',
+    updatedAt: '2018/02/22',
+  },
+  errors: [],
+  metaData: {},
+};
+
+export const BODY_ERROR =
+
+{
+  payload: {},
```

Comment:
With makeRequest we only need the payload, not the entire response. Please remove payload, errors and metaData level